aim/sdk/repo.py

```diff
@@ -192,10 +192,10 @@ def _get_container(
         if container is None:
             path = os.path.join(self.path, name)
             if from_union:
-                container = UnionContainer(path, read_only=read_only)
+                container = RocksUnionContainer(path, read_only=read_only)
```

Comment:
@alberttorosyan This is a great idea, (2) especially to prepare the codebase before the stable v3.0.0 possible to (dynamically) use different containers.
For example, to have a more optimized `Container` implementation for `seqs` / `metrics`, and the newer versions will autodetect the container format (instead of using a certain implementation based on the `VERSION`)

As we only touch the SDK as part of our renaming process, let's skip these modifications (2) and refactor them in a different PR.

Reply:
Marking this as resolved. Will have a separate conversation and task for this.